### PR TITLE
Better About Dialog

### PR DESCRIPTION
**Pull request by xStaBUx**
# Visual improvements for the About dialog:
java_5D5vrvKkJe
- [X] I have read the [contribution guidelines](https://github.com/Anuken/Mindustry/blob/master/CONTRIBUTING.md).
- [X] I have ensured that my code compiles, if applicable.
- [x] I have ensured that any new features in this PR function correctly in-game, if applicable.

### CODE REVIEW

File: core/src/mindustry/ui/dialogs/AboutDialog.java [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
 import arc.struct.*;
 import arc.util.*;
 import mindustry.game.EventType.*;
 import mindustry.gen.*;
+import mindustry.gen.Icon;
 import mindustry.graphics.*;
 import mindustry.ui.*;
 import mindustry.ui.Links.*;
 
+import javax.swing.*;
```
Comment (Anuken) [inline core/src/mindustry/ui/dialogs/AboutDialog.java:17]: oh dear

File: core/src/mindustry/ui/dialogs/AboutDialog.java [hunk 2]
```diff
@@ -1,17 +1,21 @@
 package mindustry.ui.dialogs;
 
 import arc.*;
+import arc.func.Cons;
```
Comment (Anuken) [inline core/src/mindustry/ui/dialogs/AboutDialog.java:4]: This doesn't follow style guidelines, you are not using wildcard imports and this very much looks like a redundant import.

File: core/src/mindustry/ui/Styles.java [hunk 3]
```diff
@@ -73,6 +73,8 @@
     grayi,
     /** Flat, square, black background. */
     flati,
+    /**Flat, square, gray background*/
+    flaty,
```
Comment (Anuken) [inline core/src/mindustry/ui/Styles.java:77]: This doesn't follow the right format. The letter at the end indicates the type of button style (i = image, t = text button, etc)  - what does "y" mean here?

**Review by Anuken — CHANGES_REQUESTED** [review]

File: core/src/mindustry/ui/Styles.java (hunk 3 above)
Comment (xStaBUx) [reply core/src/mindustry/ui/Styles.java:77]: Maybe I will give it the name of the panelі, because it's a gray panel with an image?

**Comment by xStaBUx** [comment]
now everything is ok?

**Comment by xStaBUx** [comment]
uh. a panel button already exists
зображення
I'd better rework my pull request

**Pull request closed without merging**